src/api/objects/MutableDomainObject.js

```diff
@@ -113,6 +113,9 @@ class MutableDomainObject {
     }
     $destroy() {
         this._observers.forEach(observer => observer());
+        //TODO: the following deletes will not work since they're not configurable properties.
+        // Also, this._globalEventEmitter doesn't seem to change after the above observer listeners are 'removed'
```

Comment:
I don't remember why `this._globalEventEmitter` is being deleted here unfortunately. It shouldn't be necessary if views etc. are cleaning up object references properly.  We could probably remove this line with no ill-effect.

Reply:
Removed.